dev/MRTCore/TestApps/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp/App.cpp

```diff
@@ -68,7 +68,7 @@ void App::OnLaunched(LaunchActivatedEventArgs const&)
     m_window = make<MainWindow>();
 
     auto window = m_window.as<winrt::winui_desktop_packaged_app_cpp::implementation::MainWindow>();
-    window->InitializeResourceLoaders(m_resourceManagerWinRT, m_resourceManagerMrm);   
+    window->InitializeResourceLoaders(m_resourceManagerWinRT/*, m_resourceManagerMrm*/);
```

Comment:
is there any harm (future work) in just deleting this second arg?

Reply:
Same reason as the header. The code is temporarily unavailable, not gone for good. I can add more comments to make that clear.